src/services/UserService.ts

```diff
@@ -40,10 +40,28 @@ const getEComStores = async (payload: any): Promise<any> => {
   });
 }
 
+const setUserPreference = async (payload: any): Promise<any> => {
+  return api({
+    url: "service/setUserPreference",
+    method: "post",
+    data: payload
+  });
+}
+
+const getUserPreference = async (payload: any): Promise<any> => {
```

Comment:
Add a TODO note here for why we are using post method for get operation

Reply:
comment Added sir